backend/substrapp/tasks/tasks.py

```diff
@@ -930,15 +930,15 @@ def save_models(subtuple_directory, tuple_type, subtuple_key):
         }
 
     elif tuple_type == COMPOSITE_TRAINTUPLE_TYPE:
-        for type_model, filename, hash_salt in [('end_head_model', OUTPUT_HEAD_MODEL_FILENAME, HASH_KEY_SUFFIX_HEAD),
-                                                ('end_trunk_model', OUTPUT_TRUNK_MODEL_FILENAME, HASH_KEY_SUFFIX_TRUNK)]:
+        for type, filename, hash_salt in [('end_head_model', OUTPUT_HEAD_MODEL_FILENAME, HASH_KEY_SUFFIX_HEAD),
```

Comment:
Using `type` is shadowing a builtin function (see https://docs.python.org/3/library/functions.html#type)

Nevertheless, it's not a reserved word (http://docs.python.org/reference/lexical_analysis.html#keywords ), but it's generally not considered as a good practice to shadow any builtin even if, here, it's reduced to the scope of the function.